portal/schema.sql

```diff
@@ -15,3 +17,13 @@ CREATE TABLE users (
     role varchar(7) NOT NULL CHECK (role IN ('teacher', 'student'))
 );
 
+-- Courses
+CREATE TABLE courses (
+  id bigserial PRIMARY KEY,
+  major text NOT NULL,
+  name text UNIQUE NOT NULL,
+  num integer UNIQUE NOT NULL,
+  description text,
+  credits integer NOT NULL,
+  teacher_id integer REFERENCES users (id)
```

Comment:
Foreign key references a `bigserial` column which could overflow this `integer` column. Teacher_id needs to be bumped up to `bigint`.